suite-native/module-passphrase/src/components/ConfirmOnDeviceBottomSheet.tsx

```diff
@@ -0,0 +1,60 @@
+import { BottomSheet, Box, Button, Text } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Translation } from '@suite-native/intl';
+
+import { DeviceTS3Svg } from '../assets/DeviceTS3Svg';
+
+const contentStyle = prepareNativeStyle(utils => ({
+    paddingTop: utils.spacings.small,
+    gap: utils.spacings.large,
+    alignItems: 'center',
+    paddingHorizontal: utils.spacings.small,
+}));
+
+const textContentStyle = prepareNativeStyle(utils => ({
+    gap: utils.spacings.small,
+}));
+
+const buttonStyle = prepareNativeStyle(_ => ({
+    width: '100%',
+}));
+
+type ConfirmOnDeviceBottomSheetProps = {
+    isVisible: boolean;
+    onClose: (isVisible: boolean) => void;
+};
+
+export const ConfirmOnDeviceBottomSheet = ({
+    isVisible,
+    onClose,
+}: ConfirmOnDeviceBottomSheetProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    const handleClose = () => onClose(false);
+
+    return (
+        <BottomSheet isVisible={isVisible} onClose={handleClose} isCloseDisplayed={false}>
```

Comment:
```suggestion
        <BottomSheet isVisible={isVisible} onClose={() => {}} isCloseDisplayed={false}>
```

But also curious why is this BottomSheet instead of Alert? Alert is better fitting component here as it requires users action to be able to progress through the flow. I guess the only reason I can find for not using alert is the Icon that we're using here is not passable to the Alert, right? 

I guess it wouldn't be too much work to allow Alerts to use either custom icon or pictogram. But if there is more reasons for this let's not overcomplicate it here and just go with BottomSheet for now.

Reply:
da93c1a3af